### DISCUSSION TITLE
Function to import CSV

### DISCUSSION THREAD

**Discussion by mixeden** [ramnes/notion-sdk-py#68] [Ideas]
I've written a function to import CSV into Notion, but I don't know if that will be OK to include it into the helpers module and open PR here.

I mean this project (notion-sdk-py) can be either "bare minimum" to communicate with Notion API or something larger.

What do you think?

**Comment by ramnes** [comment]
We all want something larger so it will be something larger, I just need to take the time to climb that one step that will make it happen, i.e. structuring the code how I want it to be and implementing a few higher level APIs. I have other priorities IRL right now but will make it happen ASAP! Feel free to open a PR to share the code and I will adapt it as soon as I'll be ready to merge such features.
